packages/data-table-server/src/dataTableService/DataTableService.ts

```diff
@@ -45,7 +45,7 @@ export abstract class DataTableService<
    */
```

Comment:
Can we add a class comment on DataTableService explaining what it is please? It only took me a few mins to grok it but would be helpful! 🙏